docs/source/api/v3/federations_all.rst

```diff
@@ -0,0 +1,93 @@
+..
+..
+.. Licensed under the Apache License, Version 2.0 (the "License");
+.. you may not use this file except in compliance with the License.
+.. You may obtain a copy of the License at
+..
+..     http://www.apache.org/licenses/LICENSE-2.0
+..
+.. Unless required by applicable law or agreed to in writing, software
+.. distributed under the License is distributed on an "AS IS" BASIS,
+.. WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+.. See the License for the specific language governing permissions and
+.. limitations under the License.
+..
+
+.. _to-api-federations-all:
```

Comment:
`WARNING: duplicate label to-api-federations-all, other instance in docs/source/api/v2/federations_all.rst.`

The label in `v2/federations_all.rst` should be `to-api-v2-federations-all`.

Reply:
Label renamed in fe141ebc8